src/backend/aspen/workflows/nextstrain_run/export.py

```diff
@@ -191,11 +192,12 @@ def write_includes_file(session, gisaid_ids, pathogen_genomes, selected_fh):
     sample_query = session.query(Sample).filter(Sample.id.in_(sample_ids))
     for sample in sample_query:
         public_identifier = sample.public_identifier
-        if public_identifier.lower().startswith("hcov-19"):
-            public_identifier = public_identifier[8:]
+        # remove leading hcov-19/ preceding characters, ignore case
+        public_identifier = re.sub(r'^hcov-19\/', "", public_identifier, flags=re.I)
         selected_fh.write(f"{public_identifier}\n")
         num_includes += 1
     for gisaid_id in gisaid_ids:
+        gisaid_id = re.sub(r'^hcov-19\/', "", gisaid_id, flags=re.I)
```

Comment:
I think we may want to do this *TOO*, but it's better to filter out the 'hcov-19' when we accept the list of gisaid ID's from the frontend... I'll add that in to this PR.

Reply:
down the road we may want to consider consolidate where strip 'hcov-19', b/c once we move over to GenBank, it will be 'SARS-CoV-2/human' all over the place......
![image](https://user-images.githubusercontent.com/20667188/146061088-ec87b2b1-e076-4977-9553-694494c533a9.png)